src/nls/it/strings.js

```diff
@@ -319,6 +318,7 @@ define({
     "CMD_FILE_SAVE_ALL"                   : "Salva tutto",
     "CMD_FILE_SAVE_AS"                    : "Salva come\u2026",
     "CMD_LIVE_FILE_PREVIEW"               : "Anteprima Live",
+    "CMD_TOGGLE_LIVE_PREVIEW_MB_MODE"     : "Abilita esperimento di Anteprima Live",
```

Comment:
By enabling this feature Live Preview go launched with your favorite browser (e.g. Firefox, IE, Opera...) and not only with Chrome.